doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -638,7 +639,6 @@ to the ``launch_hard_rt_vm.sh`` script before launching it:
       -s 2,passthru,02/0/0 \
       -s 3,virtio-console,@stdio:stdio_port \
       -s 8,virtio-net,tap0 \
-      $pm_channel $pm_by_vuart \
```

Comment:
This line should be removed in the script directly too.

Reply:
@NanlinXie will submitted another PR to delete it in the script;

Reply:
OK, thanks